sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/docgeneration/DocumentTemplateUploadLayout.java

```diff
@@ -168,17 +173,21 @@ private static final class DocumentTemplateInfoData {
 		private final String infoTextKey;
 
 		public static DocumentTemplateInfoData forDocumentTemplate(String caption, String fileName) {
-			return new DocumentTemplateInfoData(caption, fileName, "SORMAS_Document_Template_Guide.pdf",
-					Strings.headingDownloadDocumentTemplateGuide,
-					Strings.infoDownloadDocumentTemplateImportGuide
-					);
+			return new DocumentTemplateInfoData(
+				caption,
+				fileName,
+				"SORMAS_Document_Template_Guide.pdf",
```

Comment:
Needs a new guide that explains the token variable placeholder requirement

Reply:
added survey token information to the Document Template guide